egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json

```diff
@@ -31571,5 +31571,247 @@
     ],
     "insight": {},
     "_comment": " "
+  },
+  "attendanceSprayOperatorsDistrictPercent" : {
+    "chartName": "DSS_HEALTH_ATTENDANCE_SPRAY_OPERATOR_DISTRICT_PERCENT",
+    "queries": [
+      {
+        "module": "COMMON",
+        "dateRefField": "attendanceLog.time",
+        "requestQueryMap": "{\"province\": \"boundaryHierarchy.province.keyword\", \"district\": \"boundaryHierarchy.district.keyword\"}",
+        "indexName": "attendance-log-index-v1",
+        "aggrQuery": "{\"size\":0,\"aggs\":{\"FILTER\":{\"filter\":{\"bool\":{\"must\":[{\"term\":{\"attendanceLog.type.keyword\":\"EXIT\"}}]}},\"aggs\":{\"Spray Operators Marked Present\":{\"terms\":{\"field\":\"boundaryHierarchy.district.keyword\",\"size\":100},\"aggs\":{\"per_day\":{\"date_histogram\":{\"field\":\"attendanceLog.time\",\"calendar_interval\":\"day\",\"format\":\"yyyy-MM-dd\"},\"aggs\":{\"individuals_with_exit_logs\":{\"terms\":{\"field\":\"attendanceLog.individualId.keyword\",\"size\":10000,\"execution_hint\":\"map\"}},\"total_individuals_with_exit_logs_per_day\":{\"value_count\":{\"field\":\"attendanceLog.individualId.keyword\"}}}},\"Count\":{\"sum_bucket\":{\"buckets_path\":\"per_day>total_individuals_with_exit_logs_per_day\"}}}}}}}}"
```

Comment:
What's the use of `individuals_with_exit_logs` ? format field in date histogram should be used with a time field like attendanceTime, what's the use of formatting a date there?

Reply:
`individuals_with_exit_logs` is used to get the unique number of individuals who has exit logs

Reply:
but I see this aggregation `individuals_with_exit_logs` being used nowhere